CHANGELOG.md

```diff
@@ -29,6 +30,44 @@ runtime), and you will instead receive transactions for your App Router SSR requ
 
 If you are on Sentry SaaS, this may have an effect on your quota consumption: Less transactions, more spans.
 
+- **- feat(nestjs): Add nest cron monitoring support (#12781)**
+
+The `@sentry/nestjs` SDK now includes a `@SentryCron` decorator that can be used to augment the native NestJS `@Cron`
+decorator to send check-ins to Sentry before and after each cron job run:
+
+```typescript
+import { Cron } from '@nestjs/schedule';
+import { SentryCron, MonitorConfig } from '@sentry/nestjs';
+import type { MonitorConfig } from '@sentry/types';
+
+const monitorConfig: MonitorConfig = {
+  schedule: {
+    type: 'crontab',
+    value: '* * * * *',
+  },
+  checkinMargin: 2, // In minutes. Optional.
+  maxRuntime: 10, // In minutes. Optional.
+  timezone: 'America/Los_Angeles', // Optional.
+};
+
+export class MyCronService {
+  @Cron('* * * * *')
+  @SentryCron('my-monitor-slug', monitorConfig)
+  handleCron() {
+    // Your cron job logic here
+  }
+}
+```
+
+### Other Changes
+
+- feat(node): Allow to pass instrumentation config to `httpIntegration` (#12761)
```

Comment:
L: Maybe also add a code example here (there is one in the PR)

Reply:
I think that PR introduces more an escape hatch or power user feature than a feature that needs an explicit callout.